templates/create-daml-app-test-resources/index.test.ts

```diff
@@ -154,6 +153,8 @@ afterAll(async () => {
   }
 });
 
+jest.setTimeout(15000);
```

Comment:
What is this needed for?

Reply:
Just because everything got a bit slower? I think most tests already have a custom timeout. Maybe consider adding it to the few that don’t have one yet instead of setting here.

Reply:
yes, things seem to bit a bit slower, especially on mac. I'm trying to increase the timeouts only for the affected tests.